plugins/enfuse/enfuse_hdr_widget.lua

```diff
@@ -0,0 +1,10 @@
+--[[
+  enfuse focus stack processor cmd
+
+  copyright (c) 2016 Bill Ferguson
+  
+]]
+
+require "plugins/enfuse/lib/libEnfuse"
+
+return libEnfuse.hdr_widget
```

Comment:
I didn't really have an idea what a plugin looked like when I started.  The first thing I built was the plugin-data file.  If the plugin is a lib/storage/action/shortcut then the files actually have some code in them.  If the plugin is a processor, then the external processor module does most of the work and I just need hooks to get the pieces.  This was one of those instances where the code pretty much wrote itself.  I did clear_GPS first, and it had two instances so I put the main code in a library.  Next I did hugin.  That was when I realized by putting most of the code in a library that I was creating a namespace for the plugin  so that it didn't interfere or get interfered with.  I'll port an exporter to a plugin, which should more code in the files.